### PR TITLE
Feature/rule registry

### PR DESCRIPTION
**Pull request by steinkel**

### CODE REVIEW

**Comment by steinkel** [comment]
Fixing this PR to allow unique keys if configuration changes per rule

File: src/Rbac/Rules/RuleRegistry.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+ * Static rule registry to allow reusing rule instances in Rbac permissions
+ */
+class RuleRegistry
+{
+    /**
+     * Rule instances array
+     *
+     * @var array
+     */
+    protected static $rules = [];
```
Comment (rochamarcelo) [inline src/Rbac/Rules/RuleRegistry.php:26]: Registry class must have a method to clear $rules, this may affect application unit tests

**Review by rochamarcelo — CHANGES_REQUESTED** [review]

File: src/Rbac/Rules/RuleRegistry.php (hunk 1 above)
Comment (steinkel) [reply src/Rbac/Rules/RuleRegistry.php:26]: fixed, thanks

Comment (rochamarcelo) [reply src/Rbac/Rules/RuleRegistry.php:26]: I don't see changes here

Comment (steinkel) [reply src/Rbac/Rules/RuleRegistry.php:26]: sorry, pushed now

**Comment by rochamarcelo** [comment]
Thanks

**Pull request merged**